src/shared/sleep-config.h

```diff
@@ -55,10 +55,15 @@ int find_hibernate_location(HibernateLocation **ret_hibernate_location);
 int can_sleep(SleepOperation operation);
 int can_sleep_disk(char **types);
 int can_sleep_state(char **types);
-int read_battery_capacity_percentage(void);
+int read_battery_capacity_percentage(sd_device *dev);
 int battery_is_low(void);
 int get_battery_discharge_rate(void);
-int put_battery_discharge_rate(int estimated_battery_discharge_rate);
-
+int put_battery_discharge_rate(int estimated_battery_discharge_rate, uint64_t system_hash_id);
```

Comment:
this is now only used internally, so make it a static helper

Reply:
fixed.